cmake/finders/FindSIMDE.cmake

```diff
@@ -0,0 +1,105 @@
+#[=======================================================================[.rst
+FindSIMDE
+---------
+
+FindModule for SIMD Everywhere
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+.. versionadded:: 3.0
+
+This module defines the :prop_tgt:`IMPORTED` target ``SIMDE::SIMDE``.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module sets the following variables:
+
+``SIMDE_FOUND``
+  True, if headers were found.
+``SIMDE_VERSION``
+  Detected version of found SIMDE Everywhere.
+
+Cache variables
+^^^^^^^^^^^^^^^
+
+The following cache variables may also be set:
+
+``SIMDE_INCLUDE_DIR``
+  Directory containing ``simde/simde-common.h``.
+
+#]=======================================================================]
+
+# cmake-format: off
+# cmake-lint: disable=C0103
+# cmake-lint: disable=C0301
+# cmake-format: on
```

Comment:
Should no longer need these.

Reply:
Removed, since indeed cmake-format was replaced with gersemi.